SCAutolib/models/authselect.py

```diff
@@ -87,3 +87,6 @@ def __exit__(self, ext_type, ext_value, ext_traceback):
             logger.error("Exception in authselect context")
             logger.error(format_exc())
         self._restore()
+
+    def cleanup():
+        ...
```

Comment:
this should be at least TODO.

Reply:
This is not necessary at all and I will remove it. Every time the authselect was set in a context manager it was restored to the original one afterwards. So I believe in cleanup there is nothing to do. Please tell me if I am wrong and if I missed something.